src/main/java/hudson/plugins/emailext/EmailExtStep.java

```diff
@@ -0,0 +1,191 @@
+        return mimeType == null ? "" : mimeType;
+    }
+
+    @DataBoundSetter
+    public void setMimeType(@CheckForNull String mimeType) {
+        this.mimeType = Util.fixNull(mimeType);
+    }
+
+    public boolean getAttachLog() {
+        return attachLog;
+    }
+
+    @DataBoundSetter
+    public void setAttachLog(boolean attachLog) {
+        this.attachLog = attachLog;
+    }
+
+    public boolean getCompressLog() {
+        return compressLog;
+    }
+
+    @DataBoundSetter
+    public void setCompressLog(boolean compressLog) {
+        this.compressLog = compressLog;
+    }
+
+    public static class EmailExtStepExecution extends AbstractSynchronousNonBlockingStepExecution<Void> {
+
+        private static final long serialVersionUID = 1L;
+
+        @Inject
+        private transient EmailExtStep step;
+
+        @StepContextParameter
+        private transient TaskListener listener;
+
+        @StepContextParameter
+        private transient Launcher launcher;
+
+        @StepContextParameter
+        private transient Run<?,?> run;
+
+        @StepContextParameter
+        private transient FilePath workspace;
+
+        @Override
+        protected Void run() throws Exception {
+            ExtendedEmailPublisher publisher = new ExtendedEmailPublisher();
+            publisher.configuredTriggers.clear();
+
+            AlwaysTrigger.DescriptorImpl descriptor = Jenkins.getInstance().getDescriptorByType(AlwaysTrigger.DescriptorImpl.class);
+            publisher.configuredTriggers.add(descriptor.createDefault());
+
+            publisher.defaultSubject = step.subject;
+            publisher.defaultContent = step.body;
+            publisher.attachBuildLog = step.attachLog;
+            publisher.compressBuildLog = step.compressLog;
+
+            if(StringUtils.isNotBlank(step.to)) {
+                publisher.recipientList = step.to;
+            }
+
+            if(StringUtils.isNotBlank(step.replyTo)) {
+                publisher.replyTo = step.replyTo;
+            }
+
+            if(StringUtils.isNotBlank(step.attachmentsPattern)) {
+                publisher.attachmentsPattern = step.attachmentsPattern;
+            }
+
+            if(StringUtils.isNotBlank(step.mimeType)) {
+                publisher.contentType = step.mimeType;
+            }
+
+            final ExtendedEmailPublisherContext ctx = new ExtendedEmailPublisherContext(publisher, run, workspace, launcher, listener);
+            final Multimap<String, EmailTrigger> triggered = ArrayListMultimap.create();
+            triggered.put(AlwaysTrigger.TRIGGER_NAME, publisher.configuredTriggers.get(0));
+            ctx.setTrigger(publisher.configuredTriggers.get(0));
+            publisher.sendMail(ctx);
+            return null;
+        }
+    }
+
+
+    @Extension
+    public static final class DescriptorImpl extends AbstractStepDescriptorImpl {
+
+        public static final String defaultMimeType = "text/plain";

```

Comment:
Looks like you [forgot to use this](https://github.com/jenkinsci/workflow-plugin/blob/master/COMPATIBILITY.md#handling-default-values)?